README.md

```diff
@@ -137,66 +154,82 @@ Linux,
 5. Now, as long you have the Django app (https://localhost:8000) running, you should be able to make requests like POST Create User and POST Authenticate.
 Click on this link to see what you should expect: https://imgur.com/hd9VB6k
 
-POST Create User will create a new user in your localhost:8000 running Django app, and making a request to POST Authenticate will authenticate whether or not that user exists.
+- `POST` Create User will create a new user in your `localhost:8000` running Django app, 
+- making a request to `POST Authenticate` will authenticate whether or not that user exists.
 
 ![screenshot of Postman environment variable setup](https://i.imgur.com/6Uq9XQp.png) 
 
 5. Now, as long you have the Django app (https://localhost:8000) running, you should be able to make requests like `POST Create User` and `POST Authenticate` by clicking on the blue "Send" button in Postman. 
 
+</details>
+
 ## Removing Everything
 
+<details>
+<summary>To remove all containers</summary>
+<br>
 To remove all containers run the following:
 
-```plain
-docker-compose rm
+```bash
+$ docker-compose rm
 ```
 
 This will leave a copy of the data volume (holding the PostgreSQL data) behind. To remove that you will need to identify and remove the data volume.
 
-```plain
-docker volume ls
+```bash
+$ docker volume ls
 
 DRIVER              VOLUME NAME
 local               django-concept_db_data
 ```
 
 Note the name of the data volume, in this case `django-concept_db_data` and delete it.
 
-```plain
-docker volume rm django-concept_db_data
+```bash
+$ docker volume rm django-concept_db_data
 ```
 
-Note: it is likely that cached copies of your container images will be retained by Docker on your local machine. This is done to speed things up if you require these images in future. To completely remove unused container images and networks, we recommend you follow Docker [pruning guide](https://docs.docker.com/config/pruning/).
+**Note:** it is likely that cached copies of your container images will be retained by Docker on your local machine. This is done to speed things up if you require these images in future. To completely remove unused container images and networks, we recommend you follow Docker [pruning guide](https://docs.docker.com/config/pruning/).
+
+</details>
 
 ## Proof-of-concept Goals
 
 A resource datastore
 
-- save resource
-- delete resource
-- update resource
-- list resource
-- search resources
+- [ ] save resource
```

Comment:
I just realized we can tick off ALL of these checkmarks, haha 🎉 So we can put an "x" next to each of them! 

The only one we can't tick off yet is  [ ] Calendar/hangouts

Reply:
Done